eks.tf

```diff
@@ -0,0 +1,95 @@
+locals {
+  map_roles = concat([
+    {
+      rolearn  = "arn:aws:iam::${data.aws_caller_identity.iam.account_id}:role/administrators"
+      username = "administrator"
+      groups   = ["administrators"]
+    },
+    {
+      rolearn  = "arn:aws:iam::${data.aws_caller_identity.iam.account_id}:role/powerusers"
+      username = "poweruser"
+      groups   = ["powerusers"]
+    },
+    {
+      rolearn  = "arn:aws:iam::${data.aws_caller_identity.iam.account_id}:role/developers"
+      username = "developer"
+      groups   = ["developers"]
+    },
+    {
+      rolearn  = "arn:aws:iam::${data.aws_caller_identity.iam.account_id}:role/readonly"
+      username = "readonly"
+      groups   = ["readonly"]
+    },
+  ], var.map_roles)
+
+  eks_oidc_issuer = trimprefix(module.eks.cluster_oidc_issuer_url, "https://")
+}
+
+
+module "eks" {
+  source        = "terraform-aws-modules/eks/aws"
+  cluster_name = var.cluster_name
+  subnets      = var.subnets
+
+  attach_worker_cni_policy = var.attach_worker_cni_policy
```

Comment:
There's no real point in consuming the official `terraform-aws-eks` module here, if you want to pass all the variables as-is. Just reference the official module in "your" Terraform, and perhaps this should pivot into an addons module instead?

Reply:
There is a point for this. From my point of view we should provide sane defaults for security groups etc. which in the case of using the "parent" module, everyone would have to do themself. It's better to provide sane defaults for some people, this also goes for some of the modules being persisted, and others not.